docs/getting_started.md

```diff
@@ -76,6 +76,12 @@ EndTime=2001-01-02T00:00
 InputPath=/path_to_model_input_folder/
 OutputPath=/path_to_model_output_folder/
 ```
+The configuration file could also contain the following optional information:
+```text
+soil_layers_thickness=/path_to_input_soilLayThick.csv_file/
+SleepDuration=30
```

Comment:
```suggestion
```

Reply:
These two keys are needed when pystemmusscope is used. This documentation page is for stemmus_scope itself.